teacher.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'Person'
+
+class Teacher < Person
+  def initialize(id, name, age, parent_permission, specialization)
+    super(id, name, age, parent_permission)
+    @specialization = specialization
+  end
+end
```

Comment:
- [ ] In the initialize constructor method, the parameters - `name` and `parent_permission` should have default values like so 👇🏾 


```ruby
def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```
Kindly fix this